frontend/app/(groups)/g/dashboard/group-selector.tsx

```
@@ -15,12 +15,14 @@ import { isContentCreator, isAuthorizedUserFaculty } from "@/lib/utils";
 import { useSession } from "next-auth/react";
 import { useMemo, useState } from "react";
 import { Badge } from "@/components/ui/badge";
+import FavoriteBorderIcon from "@mui/icons-material/FavoriteBorder";
```

ESLint

[error] 'FavoriteBorderIcon' is defined but never used.

(@typescript-eslint/no-unused-vars)